boards/weact/bluepillplus_ch32v203/doc/index.rst

```diff
@@ -0,0 +1,87 @@
+.. zephyr:board:: bluepillplus_ch32v203
+
+Overview
+********
+
+The `WeActStudio`_ Blue Pill Plus CH32V203 hardware provides support for QingKe 32-bit RISC-V4B
```

Comment:
nit: It's the "QingKe V4B 32-bit RISC-V processor".

Reply:
Done